flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphDeploymentTest.java

```diff
@@ -742,25 +731,11 @@ private ExecutionGraph createExecutionGraph(Configuration configuration) throws
                                 0),
                         null));
 
-        final Time timeout = Time.seconds(10L);
-        return ExecutionGraphBuilder.buildGraph(
-                null,
-                jobGraph,
-                configuration,
-                executor,
-                executor,
-                new ProgrammedSlotProvider(1),
```

Comment:
Seems like this was the only remaining usage of the `ProgrammedSlotProvider`.

Reply:
Oh yes, good catch. I'll remove it :-)